tests/test_halo.py

```diff
@@ -441,6 +442,25 @@ def test_animation_setter(self):
         spinner.animation = "marquee"
         self.assertEquals("marquee", spinner.animation)
 
+    def test_spinner_color(self):
+        """Test ANSI escape characters are present
+        """
+        colors = {
+            'cyan': Fore.CYAN,
+            'magenta': Fore.MAGENTA,
+            'yellow': Fore.YELLOW,
+            'blue': Fore.BLUE
+        }
+
+        for k, v in colors.items():
+            self._stream = io.open(self._stream_file, 'w+')  # reset stream
+            spinner = Halo(color=k, stream=self._stream)
+            spinner.start()
+            spinner.stop()
+
+            output = self._get_test_output(no_ansi=False)
+            assert v in output[1]
```

Comment:
@eoinnoble We should do this here:

```suggestion
            self.assertEquals(color_ascii, output[1])
```